docs/marine-book/quick-start/setting-up-the-development-environment.md

```diff
@@ -15,15 +15,15 @@ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 # add Rust tools to the current PATH
 source $HOME/.cargo/env
 
-# install the nightly toolchain (the x86_64 suffix is needed for compatibility with M1) 
-rustup install nightly-x86_64
+# install the nightly toolchain (the x86_64 suffix is needed for compatibility with M1, and 2022-09-15 because newer rust generates wasm we temporarily do not support) 
+rustup install nightly-2022-09-15-x86_64
 ```
 
 To be able to compile Rust in Wasm, install the `wasm32-wasi` compilation target:
 
 ```sh
 # install wasm32-wasi target for WebAssembly
-rustup +nightly-x86_64 target add wasm32-wasi
+rustup +nightly-2022-09-15-x86_64 target add wasm32-wasi
```

Comment:
That's not gonna work unless a user has _this exact_ toolchain version installed. Better use `+nightly-x86_64`, cuz it refers to _any_ `nightly-x86_64`.